contributors/design-proposals/autoscaling/vertical-pod-autoscaler.md

```diff
@@ -44,7 +44,7 @@ Introduction
 
 ### Background ###
 * [Compute resources](https://kubernetes.io/docs/user-guide/compute-resources/)
-* [Resource QoS](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-qos.md)
+* [Resource QoS](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/resource-qos.md)
```

Comment:
@keni7385 Thanks for catching this!

Can you update this to use relative links i.e. `/contributors/design-proposals/node/resource-qos.md`?

Reply:
Sure. There is another absolute link pointing to a file of this repository (`https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/admission_control_extension.md#future-work`). It can be updated to `/contributors/design-proposals/api-machinery/admission_control_extension.md#future-work` as well, isn't it?

Reply:
Yes :+1:

Reply:
Done